runtime/codert_vm/cnathelp.cpp

```diff
@@ -2731,6 +2731,20 @@ old_slow_jitThrowArrayIndexOutOfBounds(J9VMThread *currentThread)
 	return setCurrentExceptionFromJIT(currentThread, J9VMCONSTANTPOOL_JAVALANGARRAYINDEXOUTOFBOUNDSEXCEPTION, NULL);
 }
 
+void* J9FASTCALL
+old_slow_jitThrowIdentityException(J9VMThread *currentThread)
+{
+	void *exception = NULL;
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	OLD_JIT_HELPER_PROLOGUE(0);
+	buildJITResolveFrameForRuntimeCheck(currentThread);
+	exception = setCurrentExceptionFromJIT(currentThread, J9VMCONSTANTPOOL_JAVALANGIDENTITYEXCEPTION, NULL);
+#else /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
```

Comment:
The comment here should be /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */

Reply:
Corrected in cfe2348588b959fc64f648aba0238aea645c7e34